sn_cli/src/subcommands/files/mod.rs

```diff
@@ -236,28 +236,20 @@ async fn upload_files(
                 error!("Failed to upload a batch: {error}");
                 recorded_upload_errors.push(error);
             } else {
-                chunk_manager.mark_paid(chunks_batch.iter().map(|(xor, _)| *xor));
-                chunk_manager.mark_verified(chunks_batch.iter().map(|(xor, _)| *xor));
+                chunk_manager.mark_completed(chunks_batch.iter().map(|(xor, _)| *xor));
             }
         }
     }
     progress_bar.finish_and_clear();
 
     // report errors
-    let failed_payments = chunk_manager.get_unpaid_chunks();
```

Comment:
I think this would not have worked prior to the changes too as we call `mark_paid and mark_verfied` right after each other.
I don't think we have any way to just know if the payment alone failed?

Reply:
Actually we do, a few lines above, we pay. Only if we fail (after all retries) we bail out.

Reply:
But you're right this code would never run indeed! Good that you removed it!

Reply:
Indeed! Although we might want to keep printing out pay errors.

Reply:
By checking `recorded_pay_errors.len()` instead

Reply:
Ah I see! Updated the PR.